public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -328,6 +318,11 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
   const rulerRuleRequests = useUnifiedAlertingSelector((state) => state.rulerRules);
   const groupfoldersForSource = rulerRuleRequests[sourceName];
 
+  const groupWithRules = getGroupFromRuler(groupfoldersForSource?.result, groupName, nameSpaceName);
+  const rulesWithoutRecordingRules: RulerRuleDTO[] =
```

Comment:
Could we reverse the names a bit? Maybe instead of saying `withoutRecording` we could use `alertingRules`? 
`const alertingRules` and then instead of `!isRecordingRulerRule` we could check `isAlertingRulerRule`

Reply:
it makes sense, I was confused with this `AlertingRules` name.